openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/DetailsDialog.vue

```diff
@@ -292,7 +292,14 @@ export default {
                     }
                   }
                 }
-                this.rawValue = values[0][0]
+                if (
+                  this.details.data_type.includes('INT') &&
+                  !this.details.array_size
+                ) {
+                  this.rawValue = '0x' + values[0][0].toString(16).toUpperCase()
+                } else {
+                  this.rawValue = values[0][0]
+                }
```

Comment:
I wanted hex to be in addition to the decimal display, not replace it.

Reply:
Something like `0xA (10)`

Reply:
That or a separate line.  Either would be fine.

Reply:
Of those: 10 (0xA)